tests/Umbraco.Tests.Integration/Umbraco.Infrastructure/Persistence/Repositories/SimilarNodeNameTests.cs

```diff
@@ -184,17 +184,21 @@ public void Test(int nodeId, string nodeName, string expected)
     }
 
     [Test]
-    [Explicit("This test fails! We need to fix up the logic")]
+    // ToDo: Check wether the Explicit attribute is still needed e.g. in pipelines
```

Comment:
I'm pretty sure this `Explicit` would have been added to remove it from running in the pipeline, so if it does actually pass both when running locally and on the pipeline build triggered from this PR, this `TODO` can be removed.